src/algebra/group/to_additive.lean

```diff
@@ -551,3 +552,34 @@ attribute [to_additive empty] empty
 attribute [to_additive pempty] pempty
 attribute [to_additive punit] punit
 attribute [to_additive unit] unit
+
+section linter
+
+open tactic expr
+
+/-- A linter that checks that multiplicative and additive lemmas have both doc strings if one of
+them has one -/
+@[linter] meta def linter.to_additive_doc : linter :=
+{ test := (λ d, do
+    let mul_name := d.to_name,
+    dict ← to_additive.aux_attr.get_cache,
+    match dict.find mul_name with
+    | some add_name := do
+      mul_doc <- doc_string mul_name >> return tt <|> return ff,
+      add_doc <- doc_string add_name >> return tt <|> return ff,
+      match mul_doc, add_doc with
+      | tt, ff := return $ some $ "declaration has a docstring, but its additive version does " ++
+        "not. You might want to pass a string argument to `to_additive`."
+      | ff, tt := return $ some $ "declaration has no docstring, but its additive version does. " ++
+        "You might want to add a doc string to the declaration."
+      | _, _ := return none
```

Comment:
Can you include the name of the corresponding lemma (that is, `add_name` and `mul_name`) in this message?

Reply:
`mul_name` would be redundant:

> Error: mul_opposite.pseudo_emetric_space - declaration has a docstring, but its additive version does not. You might want to pass a string argument to `to_additive`

But I can include the addictive name